js/app/historyStore.js

```diff
@@ -91,9 +92,11 @@ const HistoryStore = new Lang.Class({
         Dispatcher.get_default().register((payload) => {
             switch(payload.action_type) {
                 case Actions.HISTORY_BACK_CLICKED:
+                    this.get_previous_item().entry_point = EntryPoints.NAV_BUTTON_CLICKED;
```

Comment:
I don't understand how that would make duplicate calls to the metrics daemon, since you can only be viewing two items at a time (one on a page, and one in a lightbox on top of the page.) Anyway, this might be a bit overinvolved for a discussion in review comments — I'll try tomorrow to poke at some code to illustrate what I mean. Hopefully that will either convince me that I'm talking nonsense, or else convince you that it will work for your use case :smile: